standup_notes/__main__.py

```diff
@@ -83,14 +96,54 @@ def copy_note(day: date):
         print(note + ' doesn\'t exist yet.')
 
 
+def copy_prev(day: date):
+    previous_days_note = get_note_name(last_weekday(day))
+    note = get_note_name(day)
+    date_of_note = "Date: " + day.strftime("%m/%d/%Y") + " \n"
+    beginning_format = "__What I did yesterday__:\n"
```

Comment:
I don't love hard-coding these formats here because if I add something to them in the template file, it doesn't get my updates here when copying previous notes. Can you read in the line from the template file that contains 'What I did' (just that in case you change yesterday to Friday or something) and then have the end_format just read everything from the line containing 'What I'm doing' onwards?

Reply:
I just pushed some changes that I think solves this problem.

Reply:
These appear to still be hardcoded, so would still get out of sync with the template file.

Reply:
Removed the hard coding